packages/terra-dynamic-grid/docs/README.md

```diff
@@ -0,0 +1,28 @@
+# Terra Dynamic Grid
+
+{insert description}
```

Comment:
For the description, you can add `The terra-dynamic-grid component provides users a way to build dynamic layouts using CSS Grids.`